src/App.tsx

```diff
@@ -1,61 +1,27 @@
 import './App.scss';
+import { useState } from 'react';
+import { Form } from './components/Form';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todos';
+import { User } from './types/Users';
 
 export const App = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <form action="/api/todos" method="POST">
-        <div className="field">
-          <input type="text" data-cy="titleInput" />
-          <span className="error">Please enter a title</span>
-        </div>
-
-        <div className="field">
-          <select data-cy="userSelect">
-            <option value="0" disabled>
-              Choose a user
-            </option>
-          </select>
-
-          <span className="error">Please choose a user</span>
-        </div>
-
-        <button type="submit" data-cy="submitButton">
-          Add
-        </button>
-      </form>
+  const [users] = useState<User[]>(usersFromServer);
```

Comment:
Consider fetching `users` asynchronously using an effect hook (e.g., `useEffect`) to handle cases where the data might not be immediately available or could change over time. This will make your application more robust and capable of handling dynamic data sources.